runner/src/server/plugins/engine/pageControllers/RepeatingFieldPageController.ts

```diff
@@ -297,12 +297,15 @@ export class RepeatingFieldPageController extends PageController {
 
     for (let i = 0; i < answers.length; i++) {
       if (typeof answers[i] === "string") {
-        const values = answers[i].split(":");
-        let multiInput = {
-          "type-of-revenue-cost": values[0].trim(),
-          value: values[1].substring(2),
+        // TODO: This is a javascript right split, we need to have a re-think about how add another answers work
+        const reversedAnswer = answers[i].split("").reverse().join("");
+        const values = reversedAnswer.split(" : ");
+        const amount = values.shift().split("").reverse().join("");
+        const description = values.join(" : ").split("").reverse().join("");
```

Comment:
I wonder in this case might a regex with match groups be simpler? e.g.
```
const regex = /(.+) : (.+)$/
const amount = string.match(regex)[2]
const description = string.match(regex)[1]
```

Reply:
@adamdavies1 we could also remove the `£` from the match group e.g.

```
(.+) : £(.+)$
```
and get rid of that substring step I think?

Reply:
Awesome job! Implemented this now